src/App.tsx

```diff
@@ -1,31 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(({ description, title }) => {
+    const normalizedTitle = title.toLowerCase();
+    const normalizedDescription = description.toLowerCase();
+    const normalizedQuery = query.trim().toLowerCase();
```

Comment:
I think you could create a helper function for normalizing the text to make your code a bit cleaner